docs/how_to/pipeline.rst

```diff
@@ -0,0 +1,41 @@
+Pipeline
+========
+
+Define functions that are used in the third-party authentication flow.

```

Comment:
Can we link docs for social auth authentication pipeline?

Reply:
https://openedx.atlassian.net/wiki/spaces/AC/pages/936247448/Authentication this one?

Reply:
This one: https://python-social-auth.readthedocs.io/en/latest/pipeline.html. We can add something like: for more information on social auth pipelines, please refer to...